tests/e2e/configuration/storage_info.py

```diff
@@ -54,13 +54,12 @@ def test_does_default_config_match():
 
     # The default value of these is dependent on the given machine.
     machine_dependent_configurations = [
-        "memory_usage",
+        "memory_res",
```

Comment:
Did we sync with the server side about this change?

Reply:
This is only for our side, on server side we didn't change anything: https://github.com/memgraph/memgraph/pull/1426